### ISSUE TITLE
Documentation refers to `self.stats`, should be `self.data.stats`

### ISSUE THREAD

**Issue by andrewbaxter** [scrapinghub/spidermon#94]
Copying examples from the documentation raises this error:
```
getattr(self.stats, 'item_scraped_count', 0)
AttributeError: 'GeneralMonitor' object has no attribute 'stats'
```

I believe these should be `self.data.stats` as per other documentation.

**Comment by manycoding** [comment]
#77

**Comment by rennerocha** [comment]
@andrewbaxter This is fixed in PR #77
